modules/form-field/react/stories/stories.tsx

```diff
@@ -6,7 +6,7 @@ import withReadme from 'storybook-readme/with-readme';
 import {Label, Hint} from '../index';
 import README from '../README.md';
 
-storiesOf('Form Field/Label', module)
+storiesOf('Components|Inputs/Label', module)
```

Comment:
It struck me as odd that this is just under `Label`, when `Hint` is under `Form Field`

Reply:
Hmm yeah I'll move this into `Form Field`